### DISCUSSION TITLE
How to using CommunityToolkit.WinUI.Converters

### DISCUSSION THREAD

**Discussion by gaobin89** [CommunityToolkit/Windows#538] [Q&A]
I installed Converters 8.1.240916 and try to using in winui3 app xaml ,but can't found name space,how to fix ? Thanks 
![image](https://github.com/user-attachments/assets/70a9b9fd-fa18-418b-8cc6-5ec2cc500b53)

**Comment by nikolayvpavlov** [comment]
Your project must target Windows SDK 10.0.22621:

    <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>

**Reply by gaobin89** [reply]
Thanks !
